apps/voting/app/src/components/VoteCard/VoteCard.js

```diff
@@ -45,66 +45,69 @@ const VoteCard = ({ vote, onOpen }) => {
   }, [voteId, onOpen])
 
   return (
-    <Card
-      onClick={handleOpen}
-      css={`
-        display: grid;
-        grid-template-columns: 1fr;
-        grid-template-rows: auto 1fr auto auto;
-        grid-gap: ${1 * GU}px;
-        padding: ${3 * GU}px;
-      `}
-    >
+    <Card onClick={handleOpen}>
       <div
```

Comment:
The changes in https://github.com/aragon/aragon-ui/pull/533 seem to have broken this component's layout. I've now shifted the grid positioning to a new inner layout, but still having problems with long descriptions not automatically line breaking cc @AquiGorka @bpierre :

<img width="367" alt="Screen Shot 2019-08-28 at 2 52 22 PM" src="https://user-images.githubusercontent.com/4166642/63857139-76b5f800-c9a3-11e9-9946-acd2d70e97ad.png">

Reply:
Adding `css='display: block'` to the card should fix this.

Reply:
This should fix it https://github.com/aragon/aragon-ui/pull/541

There shouldn’t be any layout difference when switching to an interactive card, so let’s see if it’s entirely fixed now, otherwise we’ll move back to using a `div` with a custom focus ring.

Reply:
Now fixed with latest aragonUI.